frontend/tests/EnvironmentSetting.test.ts

```diff
@@ -1,5 +1,5 @@
 import { it, describe, expect, test } from 'vitest';
-import { EnvironmentSettings } from './EnvironmentSettings';
+import { EnvironmentSettings } from '../src/services/EnvironmentSettings';
```

Comment:
Maybe it is good to change import: '@/services/EnvironmentSettings'. Please check if test can be run.

Reply:
Thanks. I have changed as suggested.